lib/utils/cli.go

```diff
@@ -154,12 +154,14 @@ func UserMessageFromError(err error) string {
 	if err != nil {
 		// If the error is a trace error, check if it has a user message embedded in
 		// it, if it does, print it, otherwise escape and print the original error.
-		if er, ok := err.(*trace.TraceErr); ok {
-			if er.Message != "" {
-				return fmt.Sprintf("error: %v", EscapeControl(er.Message))
-			}
+		if err, ok := err.(*trace.TraceErr); ok && err.Message != "" {
```

Comment:
No need to use another name, the `err` will shadow `err` in outer scope and not overwrite it.